src/server/scripts/World/npc_innkeeper.cpp

```diff
@@ -4,17 +4,13 @@
 #include "ScriptedCreature.h"
 #include "ScriptedGossip.h"
 #include "ScriptMgr.h"
-#include "WorldSession.h"
 
-enum eTrickOrTreatSpells
-{
-    SPELL_TRICK                 = 24714,
-    SPELL_TREAT                 = 24715,
-    SPELL_TRICKED_OR_TREATED    = 24755,
-    HALLOWEEN_EVENTID           = 12,
-    GOSSIP_MENU                 = 9733,
-    GOSSIP_MENU_EVENT           = 342
-};
+constexpr auto SPELL_TRICK = 24714;
```

Comment:
Sucks we have to do this to a lot.. are we sure it doesnt work with enum class?

Reply:
yep, I've tried several times